Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/OldOneClimbyBoi.java

```diff
@@ -0,0 +1,67 @@
+//package org.usfirst.frc.team5493.robot.subsystems;
+//
+//import org.usfirst.frc.team5493.robot.Robot;
+//import org.usfirst.frc.team5493.robot.RobotMap;
+//
+//import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+//
+//import edu.wpi.first.wpilibj.CounterBase.EncodingType;
+//import edu.wpi.first.wpilibj.Encoder;
+//import edu.wpi.first.wpilibj.command.Subsystem;
+//
+///**
+// *
+// */
+//public class OldOneClimbyBoi extends Subsystem {
+//
+//	public WPI_TalonSRX climbMotor;
+//	public Object climbDown;
+//	
+//	
+//	//Encoder numbers to be added
+//	//Encoder A Channel -> DIO 1, Module 1;
+//	
+//	//private Encoder encoder = new Encoder(1, 2, false, EncodingType.k2X);
+//	
+//	
+//	public void robotInit() {
+//		//encoder.setDistancePerPulse(kDistancePerPulse);
+//		
+//		//encoder.start();
+//		
+//		resetEncoders();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)